audonnx/core/model.py

```diff
@@ -46,13 +47,31 @@ class Model(audobject.Object):
     to see all available methods.
 
     Args:
-        path: path to model file
+        path: ONNX model or path to model file
```

Comment:
What is an ONNX model? The corresponding Python object as provided by `onnxruntime`?
What's the difference between the `onnx` and `onnxruntime` package?

Reply:
It might also be that we refered with `ONNX model` to the ONNX file, because when I look at the tests you added it seems that you could load the model from the ONNX file or the model folder (and most likely also the YAML file).
This means we now have four options to provide the "path" to `audonnx.Model`.

So maybe we change to

```
    Args:
        model: ONNX model object or path to ONNX or YAML model file or the model folder
```

Reply:
Loading from YAML or a folder is not supported, therefore we have `audonnx.load()`. It also doesn't make sense to load from a YAML since we already store labels and transforms there and those could clash with the arguments the user provides in the init function.

We can also not easily rename the argument to `model` since for all our old models the argument name stored in YAML is `path`, which also makes sense cause once you store the model to YAML the ONNX file will be saved.

So I propose we change to:

![image](https://user-images.githubusercontent.com/10383417/172789988-e6f369e4-fa91-4c2e-8324-c46595b3295c.png)

Reply:
Thanks for the clarification.

I agree with your proposal.